docs/guides/nrfconnect_examples_dfu.md

```diff
@@ -0,0 +1,107 @@
+
+    That operation may take few minutes and should be finished when progress bar will reach 100%, as presented below:
+
+        565.25 KiB / 565.25 KiB [========================================================================] 100.00% 2.07 KiB/s 4m32s
+        Done
+
+6. Obtain a list of images present in the device memory, by invoking following command:
+
+        $ sudo mcumgr --conntype ble --connstring ctlr_name=ble-controller-name,peer_name='ble-device-name' image list
+
+    The displayed output contain the old image in slot 0 that is currently active and the new image in slot 1, which is not active yet (flags field empty):
+
+        Images:
+        image=0 slot=0
+            version: 0.0.0
+            bootable: true
+            flags: active confirmed
+            hash: 7bb0e909a846e833465cbb44c581cf045413a5446c6953a30a3dcc2c3ad51764
+        image=0 slot=1
+            version: 0.0.0
+            bootable: true
+            flags: 
+            hash: cbd58fc3821e749d3abfb00b3069f98c078824735f1b2a333e8a1579971e7de1
+        Split status: N/A (0)
+
+7. Swap firmware images, by calling the following method with `image-hash` replaced by the image present in the slot 1 hash (e.g.cbd58fc3821e749d3abfb00b3069f98c078824735f1b2a333e8a1579971e7de1):
+
+        $ sudo mcumgr --conntype ble --connstring ctlr_name=ble-controller-name,peer_name='ble-device-name' image test image-hash
+
+    You can observe that in image present in slot 1 `flags:` field should change value to `pending`:
+
+        Images:
+        image=0 slot=0
+            version: 0.0.0
+            bootable: true
+            flags: active confirmed
+            hash: 7bb0e909a846e833465cbb44c581cf045413a5446c6953a30a3dcc2c3ad51764
+        image=0 slot=1
+            version: 0.0.0
+            bootable: true
+            flags: pending
+            hash: cbd58fc3821e749d3abfb00b3069f98c078824735f1b2a333e8a1579971e7de1
+        Split status: N/A (0)
+
+8. Reset the device with the following command to let bootloader swap images:
+
+        $ sudo mcumgr --conntype ble --connstring ctlr_name=ble-controller-name,peer_name='ble-device-name' reset
+
+    You should be able to see that device was reset and the following logs appeared on its console:
+
+        *** Booting Zephyr OS build zephyr-v2.5.0-1101-ga9d3aef65424  ***
+        I: Starting bootloader
+        I: Primary image: magic=good, swap_type=0x2, copy_done=0x1, image_ok=0x1
+        I: Secondary image: magic=good, swap_type=0x2, copy_done=0x3, image_ok=0x3
+        I: Boot source: none
+        I: Swap type: test
+
+    Swapping operation may take some time, but after that new firmware will be booted. Please note that first firmware boot is the test boot to check integrity and allow for image revert in case of problems, but after another hardware reset firmware will be booted in the normal way.
```

Comment:
What to do if after the first reset I want to back to the previous image?

Reply:
We will not going to support that, as it would complicate our example too much. I'm going to remove that sentence or rephrase.